packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -352,6 +355,44 @@ class TransactionService {
     )
   }
 
+  setSession(tr) {
+    let session = this._config.get('session')
+
+    if (session) {
```

Comment:
nit: too much nesting, it's good to return if session is false.

Reply:
Maybe this is a difference in style, but I think this is more clear compared to a return at the top. Let's keep it for now.